polyfill/lib/plaindate.mjs

```diff
@@ -133,8 +133,8 @@ export class PlainDate {
     if (!props) {
```

Comment:
After we add monthCode/era/eraYear, will those fields be added to `['day', 'month', 'year']` in the call to `ES.CalendarFields` 2 lines above here?  I'm asking because calendars will need to know whether their `fields` implementations must add those fields or not.

Reply:
I'm assuming that `monthCode` should be added to that line since it's part of all calendars and is present in implementations without Intl. `era` and `eraYear` should not, since they aren't present on the ISO calendar and aren't present in implementations without Intl.

Reply:
(commented above, @ `ES.CalendarMergeFields` implementation; commenting here for reference)